rbx_types/src/attributes/reader.rs

```diff
@@ -0,0 +1,182 @@
+// Thanks to Anaminus! https://github.com/RobloxAPI/rbxattr/blob/master/spec.md
+
+use super::*;
+use crate::{basic_types::*, brick_color::BrickColor, variant::Variant};
```

Comment:
Can we spell out these globs? It's hard to tell where a type comes from, especially when there are multiple globs.